step.yml

```diff
@@ -51,300 +49,218 @@ toolkit:
-        when the compilation before performing the tests takes too long.
-
-        This is fixed by running `xcodebuild OPTIONS build test OPTIONS` instead of `xcodebuild OPTIONS test OPTIONS`.
-        Calling an explicit build before the test results in the code being compiled twice, thus creating an overhead.
-
-        Unless you are sure that your configuration is not prone to this error, it is recommended to leave this option turned on.
-      value_options:
-        - "yes"
-        - "no"
-      is_required: true
-  - should_retry_test_on_fail: "no"
-    opts:
-      category: Debug
-      title: "Should retry tests on failure? (Not available in Xcode 13+)"
-      summary: If you set this input to `yes`, the Step will rerun your failed tests.
-      description: |-
-        If you set this input to `yes`, the Step will rerun ALL your tests once in the case of failed test/s. Note that ALL your tests will be rerun, not just the ones that failed.
-
-        This input is not available if you are using Xcode 13+. In that case, we recommend using the `retry_on_failure` Test Repetition Mode (test_repetition_mode).
-      value_options:
-        - "yes"
-        - "no"
-      is_required: true
-  - xcpretty_test_options: --color --report html --output "${BITRISE_DEPLOY_DIR}/xcode-test-results-${BITRISE_SCHEME}.html"
-    opts:
-      category: Debug
-      title: Additional options for `xcpretty` test call
-      description: |-
-        Options added to the end of the `xcpretty` test call.
-
-        If you leave empty this input, xcpretty will be called as:
-
-        `set -o pipefail && XCODEBUILD_TEST_COMMAND | xcpretty`
-
-        In case of leaving this input on default value:
-
-        `set -o pipefail && XCODEBUILD_TEST_COMMAND | xcpretty --color --report html --output "${BITRISE_DEPLOY_DIR}/xcode-test-results-${BITRISE_SCHEME}.html"
-
-        If you want to add more options, list that separated by space character.
-  - cache_level: swift_packages
-    opts:
-      category: Debug
-      title: Enable caching of Swift Package Manager packages
-      description: |-
-        Available options:
-        - `none` : Disable caching
-        - `swift_packages` : Cache Swift PM packages added to the Xcode project
-      value_options:
-      - "none"
-      - "swift_packages"
-  - collect_simulator_diagnostics: "never"
-    opts:
-      category: Debug
-      title: Collect Simulator diagnostics
-      value_options:
-      - "always"
-      - "on_failure"
-      - "never"
+- project_path: $BITRISE_PROJECT_PATH
+  opts:
+    title: "Project path"
+    summary: Xcode Project (`.xcodeproj`) or Workspace (`.xcworkspace`) path.
+    description: |-
+      Xcode Project (`.xcodeproj`) or Workspace (`.xcworkspace`) path.
+
+      The input value sets xcodebuild’s `-project` or `-workspace` option.
```

Comment:
the usage of `’` and `'` apostrophes are mixed in the step.yml, I suggest unifying it by using `'` only.

Reply:
@Bence1001 2 italic apostrophes remained in the step.yml/description (line 16), pls fix them too.